code/modules/jobs/job_types/geneticist.dm

```diff
@@ -9,6 +9,8 @@
 	supervisors = "the chief medical officer and research director"
 	selection_color = "#d4ebf2"
 	chat_color = "#83BBBF"
+	minimal_player_age = 1
+	exp_requirements = 120
```

Comment:
geneticists have almost no IC obligations. I don't think they should have any restrictions, their capacity to grief is quite low too

Reply:
Hulk smash.

Reply:
It feels like this PR is ignoring the fact that people server hop. Anybody moving from a different SS13 server is going to see very few jobs available until they play quite a few hours here. 

Grief protection is kind of like DRM on games - you get delay a few bad apples but eventually they'll be a problem again, plus it degrades the experience for all regular users.

Reply:
Hulk griefing is one of the things that is done by older players not 0h grief accounts. They go for the more damaging route of just bombing the station.

Reply:
Yeah, hulk isn't something you can quickly mass grief with. Not without an admin spotting it. I strongly believe geneticist should have little to no time requirement